x/leveragelp/client/cli/query_committed_tokens_locked.go

```diff
@@ -0,0 +1,46 @@
+package cli
+
+import (
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/elys-network/elys/x/leveragelp/types"
+	"github.com/spf13/cobra"
+)
+
+var _ = strconv.Itoa(0)
+
+func CmdCommittedTokensLocked() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "committed-tokens-locked [address]",
+		Short: "Query committed-tokens-locked",
```

Comment:
- Please add something descriptive maybe `This query shows committed tokens for all the position for a address`
- I think we have similar query in commitment module but that one displays for one address only.

Reply:
Thank you. It was changed.